cdap-docs/admin-manual/source/operations/preferences.rst

```diff
@@ -0,0 +1,75 @@
+.. meta::
+    :author: Cask Data, Inc.
+    :copyright: Copyright © 2014-2015 Cask Data, Inc.
+
+.. _preferences:
+
+=================================
+Preferences and Runtime Arguments
+=================================
+
+.. include:: ../../../_common/_include/include-v260-deprecate-procedures.rst
```

Comment:
As you have no mention of procedures in this file, we could drop the deprecation notice.